pt/deploy/README.md

```diff
@@ -193,14 +194,12 @@ Assim como fez em seu próprio computador, você pode criar um virtualenv na Pyt
 
     20:20 ~ $ source myvenv/bin/activate
 
-    (mvenv)20:20 ~ $  pip install django whitenoise
+    (mvenv)20:20 ~ $  pip install -r requirements.txt
```

Comment:
@scorphus There's a freeze on this translation. On the next pull requests, we should start the translation to fit the en version using the 1.10 version.